doc/theory/sh_basis.rst

```diff
@@ -57,28 +68,42 @@ literature for the computation of the ODF. DIPY implements two of these in the
 
     Y_i(\theta, \phi) =
      \begin{cases}
-     \Im(Y_l^{|m|}(\theta, \phi)) & -l \leq m < 0, \\
+     \Im(Y_l^{m}(\theta, \phi)) & -l \leq m < 0, \\
      Y_l^0(\theta, \phi) & m = 0, \\
      \Re(Y_{l}^m(\theta, \phi)) & 0 < m \leq l
      \end{cases}
 
 In both cases, $\Re$ denotes the real part of the spherical harmonic basis, and
-$\Im$ denotes the imaginary part.
+$\Im$ denotes the imaginary part. The SH bases are both orthogonal and real. Moreover,
+the `descoteaux07` basis is orthonormal. 
 
-In practice, a maximum even order $k$ is chosen such that $k \leq l$. By only
+The basis proposed by Tournier is the one used in MRtrix [3]_. However, the MRtrix
+implementation differs from the cited paper definition, as it is given by:
+
+..  math::
+
+    Y_i(\theta, \phi) =
+     \begin{cases}
+     \sqrt{2}\Im(Y_l^{|m|}(\theta, \phi)) & -l \leq m < 0, \\
+     Y_l^0(\theta, \phi) & m = 0, \\
+     \sqrt{2}\Re(Y_{l}^m(\theta, \phi)) & 0 < m \leq l
+     \end{cases}
+
+As we can see, the differences include a $\sqrt{2}$ factor multiplying the SH functions
+when $m$ is different than 0 as well as an absolute value for the degree $m$ when $m$ is negative.
+The $\sqrt{2}$ factor has been added to the definition in MRtrix3 in order to make the Tournier basis
+orthonormal. Considering the absolute value of $m$ when $m < 0$ only affects the sign of the
+resulting SH functions, due to the relations $-p=|p|$ $\forall p < 0$ and 
+$Y_l^{-m} = (-1)^m\overline{Y_l^{m}}$, and therefore still results in a valid basis.
+
+In practice, a maximum even order $k$ is used to truncated the SH series. By only
```

Comment:
`is used to truncated the SH series.`: `is used to truncate the SH series.`

Reply:
Hi, the comments regarding sh_basis.rst have been adressed in commits d928b33  and 3f9ea83 !

Reply:
Addressed in d928b33.